nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -94,7 +94,7 @@ The default [Critical Error Action](/nservicebus/hosting/critical-errors.md) for
 
 snippet:DefaultHostCriticalErrorAction
 
-WARNING: It is important to consider the effect these defaults will have on other things hosted in the same process. For example if co-hosting NServiceBus with a web-service or website.
+It is not required to override this callback. However, if it is required to execute code just before shutdown then it can be overridden in the same way as when self hosting. See [When to override the default critical error action](/nservicebus/hosting/critical-errors.md#when-to-override-the-default-critical-error-action) for guidance.
```

Comment:
> It is not required to override this callback.

So in this case we won't have the problem with stopping the endpoint without stopping the process? If so, we should make clear in the hosting article which hosting options have this problem, as it's not clear.